components/nimbus/src/evaluator.rs

```diff
@@ -316,6 +320,58 @@ mod tests {
             })
         ));
     }
+    use std::collections::HashMap;
+
+    #[test]
+    fn test_targeting_app_specific_context() {
+        // Here's our valid jexl statement
+        let expression_statement =
+            "app_id == '1010' && (app_version == '4.4' || locale == \"en-US\") && is_first_run == 'true' && ios_version == '8.8'";
```

Comment:
I had only just noticed that all the existing attributes were `String`. Question: does JEXL allows us to get numbers from strings? If so, this should definitely be documented/tested here.

Reply:
I'm pretty sure it does, since we can use comparison operators..   

See:
https://github.com/mozilla/jexl-rs/blob/351aaeb6eb172b2538a46a6aa52cc14bd1b0d909/jexl-parser/src/ast.rs#L47

and looking at some of the tests in jexl-rs

Reply:
I'll create a follow-up ticket in the testing epic to add more comprehensive jexl tests!